roles/scm/add-webhooks-github/tests/inventory/hosts

```diff
@@ -0,0 +1,2 @@
+[webhooks-server]
+localhost
```

Comment:
Probably should use `ansible_connection=local`, so please add a `host_vars` file for `localhost.yml` and add this.